frontend/src/Student/components/OtherMediaSubmissionForm.js

```diff
@@ -226,7 +229,9 @@ class OtherSubmissionForm extends Component {
                       participants: values.groupParticipants
                     }
                     : null,
-                hometown: values.hometown,
+                hometown: values.submittingAsGroup === 'no'?  
+                  values.hometown
+                  : null,
```

Comment:
Only submits a value for hometown if its an individual submission

Reply:
I think Formik is null tolerant enough that this is what happens anyway without the ternary, but I like that this is explicit about it.

Reply:
this is how they did the other group submission fields, I figure its good to keep it consistent if nothing else